pandas/io/pytables.py

```diff
@@ -95,10 +95,10 @@ def _ensure_term(where, scope_level):
                 wlist.append(w)
             else:
                 wlist.append(Term(w, scope_level=level))
-        where = wlist
+        where = wlist if wlist else None
```

Comment:
this is not needed

Reply:
This is connected with next request. Check it please.